src/TNBase/Forms/Printing/FormPrintBirthdays.cs

```diff
@@ -140,13 +140,38 @@ private void btnCancel_Click(object sender, EventArgs e)
 
         private void formPrintBirthdays_Load(object sender, EventArgs e)
         {
-            printBirthdaysForm();
-            this.Close();
+            DateTime fromDate;
+            DateTime toDate;
+
+            serviceLayer.GetUpcomingBirthdayDates(out fromDate, out toDate);
+
+            dtpFrom.Value = fromDate;
+            dtpTo.Value = toDate;
         }
 
         public FormPrintBirthdays()
         {
             InitializeComponent();
         }
+
+        private void btnPrint_Click(object sender, EventArgs e)
+        {
+            pnlParams.Visible = false;
+            lblPrinting.Text = string.Format(lblPrinting.Text, dtpFrom.Text, dtpTo.Text);
+            pnlPrinting.Visible = true;
+            Refresh();
+            printBirthdaysForm();
```

Comment:
Not sure where it is set but the printer report title needs to reflect the correct date as well:
![image](https://user-images.githubusercontent.com/7377062/216725693-12bb3e65-9c34-4637-af3c-0fbeee2b9339.png)